CONTRIBUTING.md

```diff
@@ -0,0 +1,122 @@
+# Contribution Guide
```

Comment:
We already have a contributing guide here? https://github.com/VSCodeVim/Vim/blob/master/.github/CONTRIBUTING.md. Why the copy?

Reply:
In the case of Korean users, I thought I would feel uncomfortable when I read the document, so I worked on Korean
I want to keep separate files. 
If you don't mind, I suggest uploading donation guides to the Korean version so that Korean users can find them in the project.